### PR TITLE
Some suggestions

### PR DESCRIPTION
**Pull request by ubaldot**
I noticed that there are several mismatch between the readme file and what is going to be installed. Also, there is a number of plugins that are not even mentioned anywhere (like git-fugitive, vim slime, etc.). 

Mind that if too much stuff got installed, people may get annoyed because Vim may slow down and/or some plugins may be in conflict. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,46 +12,75 @@
...
+Invoke-WebRequest
+https://raw.githubusercontent.com/hungpham3112/vide/main/bin/install.ps1
+-UseBasicParsing | Invoke-Expression
 ```
 
 ### Linux (Ubuntu-Based)
 
 ```sh
-bash <(curl -s https://raw.githubusercontent.com/hungpham3112/vide/main/bin/install.sh)
+bash <(curl -s
```
Comment (hungpham3112) [inline README.md]: Take a look at this

File: README.md [hunk 2]
```diff
@@ -12,46 +12,75 @@
...
+In the future, it will be continue to compatible with `MacOS` and more `Linux`
+distro.
 
 ## :rocket: Quick-start
 
 ### Windows (Powershell 5.1+)
 
 ```powershell
-Invoke-WebRequest https://raw.githubusercontent.com/hungpham3112/vide/main/bin/install.ps1 -UseBasicParsing | Invoke-Expression
+Invoke-WebRequest
```
Comment (hungpham3112) [inline README.md]: This code will fail

**Comment by hungpham3112** [comment]
Fix your code. I know you want to suggest your plugin in here, but take a look at line warp. 
Maybe you are setting in your code `set textwidth= 80` but my code equal 120. That the reason why README.md is broken. Thanks

**Comment by ubaldot** [comment]
Oh damn! Correct! I have 80 lines textwidth, you are correct. I am going to fix it!

**Comment by hungpham3112** [comment]
> I noticed that there are several mismatch between the readme file and what is going to be installed. Also, there is a number of plugins that are not even mentioned anywhere (like git-fugitive, vim slime, etc.).

If you read the last line in the table, I include `Many more...` in there.

> Mind that if too much stuff got installed, people may get annoyed because Vim may slow down and/or some plugins may be in conflict. :)

I tested many times and all my setting is totally compatible in Windows. It still needs to check more. That is the reason why I need a person to test in other platform like MacOS.

**Comment by ubaldot** [comment]
> 
> If you read the last line in the table, I include `Many more...` in there.
> 

True! But as a user I would like to know at least the number of plugins that I am going to install (~10, ~50, ~100?). I think this information should be at least provided. 

> I tested many times and all my setting is totally compatible in Windows. It still needs to check more. That is the reason why I need a person to test in other platform like MacOS.

I don't mean compatibility, but speed. If you believe that in-spite of all these plugins Vim is still very snappy then it is ok! :)

**Comment by hungpham3112** [comment]
> 

> True! But as a user I would like to know at least the number of plugins that I am going to install (~10, ~50, ~100?). I think this information should be at least provided.
I agree, as a new user, I will need verbose information too.
> > I tested many times and all my setting is totally compatible in Windows. It still needs to check more. That is the reason why I need a person to test in other platform like MacOS.
> 
> I don't mean compatibility, but speed. If you believe that in-spite of all these plugins Vim is still very snappy then it is ok! :)

sorry for my obscure words, `compatible` in here means all the plugin can work together with the hyper-workflow in Windows, not about Vim version :)). We all need speed, we will take a look at each new plugin and it behavior in whole system.

I will create some new branch for each new plugin or thing we want to put in here, e.g: vim-yank, file-search...

If you want to use new plugin that you want to or new setting. You can put it in the file named `user_config.vim`. That is the file I created for other user who want to use their own config.

**Comment by hungpham3112** [comment]
Sorry, I changed the name branch then it close the issue

**Comment by ubaldot** [comment]
No worries :)

**Comment by hungpham3112** [comment]
> I noticed that there are several mismatch between the readme file and what is going to be installed. Also, there is a number of plugins that are not even mentioned anywhere (like git-fugitive, vim slime, etc.).


> Mind that if too much stuff got installed, people may get annoyed because Vim may slow down and/or some plugins may be in conflict. :)

You can use `:StartupTime` to measure loading time. In windows currently is around <400ms and in Linux it will be faster.
image

Worrying about plugin config I optimized config many time to reduce config as much possible, you can see older commit for more detail. You are in the same philosophy with me, `plugin compatible` is one of the most important thing in this project.
Just test it in your normal workflow and tell me if anything missing in this repo.

**Pull request closed without merging**